src/sparseml/pytorch/image_classification/integration_helper_functions.py

```diff
@@ -27,17 +27,46 @@
 )
 
 
-def create_model(source_path: Union[Path, str], **kwargs) -> torch.nn.Module:
+def create_model(
+    source_path: Union[Path, str], **kwargs
+) -> Tuple[torch.nn.Module, Dict[str, Any]]:
     """
     A contract to create a model from a source path
 
     :param source_path: The path to the model
-    :return: The torch model
+    :param kwargs: Additional kwargs to pass to the model creation function
+    :return: A tuple of the
+        - torch model
+        - additional dictionary of useful objects created during model creation
     """
-    model, *_ = create_image_classification_model(checkpoint_path=source_path, **kwargs)
-    return model
+    model, *_, validation_loader = create_image_classification_model(
+        checkpoint_path=source_path, **kwargs
+    )
+    return model, dict(validation_loader=validation_loader)
+
+
+def create_dummy_input(
+    validation_loader: Optional[torch.utils.data.DataLoader] = None,
```

Comment:
how do we expect this optional value to flow from the create model function? thinking we need to accept `**kwargs` in this signature to avoid edge cases for extra kwargs (ie arch key that we'll need)

Reply:
```python

# def [helper_functions].create_model(...) -> Tuple[torch.nn.Module, Dict[str, Any]]

model: torch.nn.Module, auxiliary_items: Optional[Dict[str, Any]] = helper_functions.create_model(...)

# in case of IC, auxiliary_items is either dict(validation_loader=validation_loader) or dict(validation_loader=None), so this should be picked up by `create_dummy_input`
```

Maybe i'd call the variable more generically, i.e. `data_loader` instead of `validation_loader`